src/masternodes/mn_checks.cpp

```diff
@@ -3541,7 +3541,7 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
                 if (paybackTokenId == loanTokenId)
                 {
                     // If interest was negative remove it from sub amount
-                    if (height >= static_cast<uint32_t>(consensus.FortCanningEpilogueHeight) && subInterest < 0)
+                    if (subInterest < 0)
```

Comment:
this will risk chain split while everyone updates, right? cause when someone uses the higher DUSD amount in the wallet  after the update, the not-yet-updated nodes will reject that block.
Any plans how to best coordinate that with everyone?

Reply:
Yes. Indeed. The majority network needs to be on v2.x until the fork block. And then switch over. We're exploring the deployment strategies before getting these merged.

Reply:
(In addition, we're also exploring fixing https://github.com/DeFiCh/ain/pull/1533 together, use this as a one-shot chance to benefit users and fix past bugs)

Reply:
really love it when this gets in. Major benefit for the community! I believe if cake & mydefichain (main ocean providers IMHO) switch coordinated, LW is fine. Probably need to get the word out to everyone with own node to have a suggested date for the update. 
Looking forward to your plan and how I can help.